components/clp-package-utils/clp_package_utils/scripts/compress.py

```diff
@@ -21,28 +25,165 @@
 logger = logging.getLogger(__file__)
 
 
-def main(argv):
-    clp_home = get_clp_home()
-    default_config_file_path = clp_home / CLP_DEFAULT_CONFIG_FILE_RELATIVE_PATH
+def _generate_targets_list(
+    container_targets_list_path: pathlib.Path,
+    parsed_args: argparse.Namespace,
+) -> None:
+    input_type = parsed_args.input_type
+
+    if InputType.FS == input_type:
+        compression_targets_list_file = parsed_args.path_list
+        with open(container_targets_list_path, "w") as container_targets_list_file:
+            if compression_targets_list_file is not None:
+                with open(compression_targets_list_file, "r") as targets_list_file:
+                    for line in targets_list_file:
+                        resolved_path = pathlib.Path(line.rstrip()).resolve()
```

Comment:
* We should still check for an empty path otherwise this will resolve to the current working directory, right?
* Can we move the operation to add the CONTAINER_INPUT_LOGS_ROOT_DIR prefix back into this script? The native scripts are (or at least were) written in such a way that they shouldn't need to know that they're running in a container whereas the non-native scripts do have to deal with all the container stuff. So I'd rather keep the `CONTAINER_INPUT_LOGS_ROOT_DIR` logic in the non-native scripts. Ditto below.